Applications/ApplicationsLib/UncoupledProcessesTimeLoop.h

```diff
@@ -29,7 +29,7 @@ template<typename Matrix, typename Vector>
 class UncoupledProcessesTimeLoop
 {
 public:
-    bool loop(ProjectData& project, std::string const& outdir);
+    bool loop(ProjectData& project);
```

Comment:
But this way the time loop can be constructed independently from the project.
Also: It has been like that before this PR.